<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="pic1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>              1</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>              2</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>               3</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>              4</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>              5</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>              6</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>              7</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>              8</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>              9</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic10">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>              10</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic11">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>              11</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic12">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>              12</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic13">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>              13</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic14">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>              14</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic15">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>              15</string>
    </property>
   </widget>
   <widget class="QPushButton" name="solve_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Собрать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mix_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Перемешать</string>
    </property>
   </widget>
  </widget>
  <action name="mix">
   <property name="text">
    <string>Перемешать</string>
   </property>
  </action>
  <action name="solve">
   <property name="text">
    <string>Собрать</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
